File: eBiblioteka.Modeli/Autor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBiblioteka.Modeli
{
    public class Autor
    {
        public int AutorId { get; set; }

        public string Ime { get; set; } = null!;

        public string Prezime { get; set; } = null!;

        public DateTime? DatumRodjenja { get; set; }

        public string? Biografija { get; set; }
    }
}

File: eBibliotekaUI/ebiblioteka_admin/lib/models/autor.dart
import 'package:json_annotation/json_annotation.dart';

part 'autor.g.dart';

@JsonSerializable()
class Autor {
  int? autorId;
  String? ime;
  String? prezime;
  DateTime? datumRodjenja;
  String? biografija;

  Autor({this.autorId, this.ime, this.prezime});

  factory Autor.fromJson(Map<String, dynamic> json) => _$AutorFromJson(json);

  Map<String, dynamic> toJson() => _$AutorToJson(this);
}
